jsm.go

```diff
@@ -462,6 +465,54 @@ func (js *js) DeleteStream(name string) error {
 	return nil
 }
 
+type apiMsgGetRequest struct {
+	Seq uint64 `json:"seq"`
+}
+
+// StreamMsg is a raw message stored in JetStream.
+type StreamMsg struct {
+	Subject  string    `json:"subject"`
+	Sequence uint64    `json:"seq"`
+	Header   []byte    `json:"hdrs,omitempty"`
```

Comment:
We should convert this if its there like a normal message no?

Reply:
Updated to decode the header and use the http.Header type.